cat/CaT-service.yaml

```diff
@@ -184,8 +184,8 @@ paths:
             application/json:
               schema:
                 oneOf:
+                  - $ref: '#/components/schemas/SalesforceProjectTender200Response'   
```

Comment:
Should this be OneOf or now we know the solution should we make it just the required response?